File: Assignment_1/Assignment1_3.py
"""--------------------------------------------------------------------------------------
                            Assignment1_3 
                    (Student name - Vaishali Jorwekar)
                     Python by Marvellous Infosystems
----------------------------------------------------------------------------------------
Problem statement :
                This program accepts two numbers from user and pass these numbers as 
                parameters to the function named Add(num1 , num2) and returns result 
                of addition 
-----------------------------------------------------------------------------------------"""
#------------------------------------------------------------------
# This function accepts two numbers as parameters and 
# returns addition of these numbers
#-------------------------------------------------------------------
def Add(num1,num2):
    result = num1 + num2
    return result
#-------------------------------------------------------------------
#-------------------------------------------------------------------
# This function accepts two numbers no1 and no2  from user and 
# calls Add(no1,no2) function
#-------------------------------------------------------------------
def main():
    print("---------------------------------------------------")
    print("Enter first number : ")
    no1 = int(input())          # Accept input and conversion to int
    print("Enter second number : ")
    no2 = int(input())          # Accept input and conversion to int
    ans = Add(no1,no2)          # Return value 
    print("-------------------------------------------------")
    print(f"Addition of {no1} and {no2} is :",ans)
    print("-------------------------------------------------")
#----------------------------------------------------------------  
#------------------------------------------------------------------  
# Entry point of program
#------------------------------------------------------------------
if __name__=="__main__":
    main()
#----------------------------------------------------------------    

File: JavaPractice/Assignment_1/Assignment1_3.java
/*--------------------------------------------------------------------------------------
                            Assignment1_3 
                    (Student name - Vaishali Jorwekar)
----------------------------------------------------------------------------------------
Problem statement :
                This program accepts two numbers from user and pass these numbers as 
                parameters to the function named Add(num1 , num2) and returns result 
                of addition 
-----------------------------------------------------------------------------------------*/
import java.util.Scanner;
public class Assignment1_3 {
    
    public int Addition(int num1,int num2){
        return num1+num2;
    }
    public static void main(String args[]){
        Scanner scanObj=new Scanner(System.in);
        System.out.println("Enter the first number:");
        int number1=scanObj.nextInt();

        System.out.println("Enter the second number:");
        int number2=scanObj.nextInt();

        Assignment1_3 addObj=new Assignment1_3();
        int result=addObj.Addition(number1, number2);
        System.out.println("Result of addition is :"+result);

        scanObj.close();

    }
}
